storage/azure/src/test/integration/backend/AzureServerStorage.test.ts

```diff
@@ -0,0 +1,112 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import "reflect-metadata";
+
+import { BlobClient, ContainerClient } from "@azure/storage-blob";
+
+import {
+  buildObjectKey,
+  buildObjectReference,
+} from "@itwin/object-storage-core/lib/common/internal";
+
+import { ObjectReference, TransferConfig } from "@itwin/object-storage-core";
+import {
+  InternalStorageIntegrationTests,
+  TestRemoteDirectory,
+  TestRemoteDirectoryManager,
+  testDeleteObjectWithTransferConfig,
+  testListObjectsWithTransferConfig,
+} from "@itwin/object-storage-tests-backend";
+
+import {
+  assertAzureTransferConfig,
+  buildBlobUrl,
+} from "../../../common/internal";
+import { AzureServerStorageBindings } from "../../../server";
+import { ServerStorageConfigProvider } from "../ServerStorageConfigProvider";
+
+describe("AzureServerStorage internal tests", () => {
```

Comment:
nitpick: we usually interpolate symbol names into tests: `${AzureServerStorage.name}`

Reply:
Updated